src/main/java/io/cdap/e2e/pages/actions/WranglerActions.java

```diff
@@ -0,0 +1,260 @@
+/*
+ * Copyright © 2023 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.cdap.e2e.pages.actions;
+
+import io.cdap.e2e.pages.locators.WranglerLocators;
+import io.cdap.e2e.utils.ElementHelper;
+import io.cdap.e2e.utils.PluginPropertyUtils;
+import io.cdap.e2e.utils.SeleniumDriver;
+import io.cdap.e2e.utils.SeleniumHelper;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.Select;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * CDF wrangler related actions.
+ */
+public class WranglerActions {
+  private static final Logger logger = LoggerFactory.getLogger(WranglerActions.class);
+
+  static {
+    SeleniumHelper.getPropertiesLocators(WranglerLocators.class);
+  }
+
+  public static void selectDirective(String columnName, String directive) {
```

Comment:
Please add javadoc comments for all the actions. You may refer to the existing Actions classes.

Reply:
added